Practical/Scientific_Calculator/index1.html

```diff
@@ -0,0 +1,158 @@
+<table class="table table-bordered ">
+  <tbody>
+    <tr>
+      <td class="text-center bg-light"><button type="button"

```

Comment:
alphabets should not be allowed in the input field. Also number should be added from the keyboard